src/main/java/org/reactome/release/qa/check/InferredModifiedResidueCheck.java

```diff
@@ -0,0 +1,54 @@
+package org.reactome.release.qa.check;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.reactome.release.qa.annotations.SliceQACheck;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QACheckerHelper;
+import org.reactome.release.qa.common.QAReport;
+
+import java.util.Collection;
+
+/**
+ *  This QA check was mostly used during the CoV-1-to-CoV-2 inference process. For now it is being kept, but may
+ *  be removed, given it was for a specific curations (August 2020).
+ *
+ *  Flags an ModifiedResidue instances that have the string '[INFERRED]' in their displayName.
+ *
+ * @author jcook
+ */
+
+@SliceQACheck
+public class InferredModifiedResidueCheck extends AbstractQACheck {
+
+    @Override
+    public QAReport executeQACheck() throws Exception {
+        QAReport report = new QAReport();
+
+        Collection<GKInstance> modifiedResidues = dba.fetchInstancesByClass(ReactomeJavaConstants.AbstractModifiedResidue);
+        for (GKInstance modifiedResidue : modifiedResidues) {
+            if (modifiedResidue.getDisplayName().contains("[INFERRED]")) {
+                report.addLine(getReportLine(modifiedResidue));
+            }
+        }
```

Comment:
Similar to above, should the modified residues be filtered so it is only those attached to instances with a Cov-2 species instead of any inferred modified residue?

Reply:
This is actually a highly specific QA test, as the Orthoinference CoV-1 project work needed some way to identify 'inferred' modified residues, and a quick solution was to just update the displayName as the inferred modified residue was created. I think the stuff said in https://github.com/reactome/release-qa/pull/26/files#r533776361 apply here though, where similar work in the future might benefit from this test existing. As such, keeping it generalized would be the way to go, in my opinion